### ISSUE TITLE
Add a Rate Us option in the navbar

### ISSUE THREAD

**Issue by bhumii-ka** [4darsh-Dev/CogniGaurd#110]
**Is your feature request related to a problem? Please describe.**
There is no option for the user to tell about his experience and give feedback.

**Describe the solution you'd like**
Adding an Rate Us page will allow user to rate the product and write some additional feedback.

Please assign this issue to me under GSSoC'24

**Comment by 4darsh-Dev** [comment]
Sure @bhumii-ka, 
you can work on creating a separate contact/rate page containing feedback  form and Rate us option. 
Are you familiar with HTML,CSS and js ?
1. Propose the design first, share SS, design or figma link.

**Comment by bhumii-ka** [comment]
@4darsh-Dev, Yes I am familiar with HTML,CSS and JS and here is the design for the page :-

![image](https://github.com/4darsh-Dev/CogniGaurd/assets/151030407/e4563e4e-7e86-4f87-a36e-04429f29dc99)
![image](https://github.com/4darsh-Dev/CogniGaurd/assets/151030407/e6056045-f61c-44ab-8374-62a124b76aa5)
![image](https://github.com/4darsh-Dev/CogniGaurd/assets/151030407/232e874f-2826-4be2-8f70-75bbd85f059e)

Please tell me if you want any changes in the design and assign this issue to me.

**Comment by 4darsh-Dev** [comment]
@bhumii-ka designs are good, 
but could be more better, use 3d animated icons and add more fields in feedback form like name 
references : https://dribbble.com/tags/feedback-form

**Comment by 4darsh-Dev** [comment]
⚠️[URGENT]⚠️ Reminder for GSSoC 24' Contributors Please join our CogniGuard Project #general GSSoC'24 Discord channel:
 Here you can discuss your queries and doubts with your mentors. 
 See #109 issues section.

**Comment by 4darsh-Dev** [comment]
⚠️[URGENT]⚠️ Reminder for GSSoC 24' Contributors Please join our CogniGuard Project #general GSSoC'24 Discord channel:
 Here you can discuss your queries and doubts with your mentors. 
 See #109 issues section.

**Comment by 4darsh-Dev** [comment]
@bhumii-ka  any work update?

**Comment by bhumii-ka** [comment]
@4darsh-Dev I'm sorry for the delay I have seen the designs in the link you sent and I will be merging those ideas with the designs I sent along with adding some 3d effects. I'll be making pr in 2 -3 days

**Comment by 4darsh-Dev** [comment]
okay @bhumii-ka 
you can talk to your mentor regarding any suggestions in project discord.

**Comment by 4darsh-Dev** [comment]
any update @bhumii-ka ?

**Comment by 4darsh-Dev** [comment]
⚠️[URGENT]⚠️ Reminder for Closing of Contribution period for GSSoC 24'
 Finish your work and raise the PRs ASAP to get your points. 
 Thanks💗 for your valuable Contributions!

**Comment by 4darsh-Dev** [comment]
GSSoC 24 has been completed Finally'
 It was a great experience working with you all 
 Thanks💗 for your valuable contributions! 
 PA nomination has been started, Do fill out the forms soon. Share your experiences and let's connect on socials
